modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPFailureTest.java

```diff
@@ -78,15 +108,6 @@ public void testFinish() {
         RestAssured.basePath = StringUtils.EMPTY;
     }
 
-    @DataProvider(name = "restAPIUserConfigProvider")
-    public static Object[][] restAPIUserConfigProvider() {
-
-        return new Object[][]{
-                {TestUserMode.SUPER_TENANT_ADMIN},
-                {TestUserMode.TENANT_ADMIN}
-        };
-    }
-
```

Comment:
Is there any reason why we bring it up.

Reply:
This is fixed with 53c5cf7c40298253d04e691e9222d21f771638fe